src/drivers/imx/sai.c

```diff
@@ -176,25 +176,30 @@ static inline int sai_set_config(struct dai *dai,
 	/* DAI clock master masks */
 	switch (config->format & SOF_DAI_FMT_MASTER_MASK) {
 	case SOF_DAI_FMT_CBS_CFS:
-		break;
-	case SOF_DAI_FMT_CBM_CFM:
+		trace_sai("SAI: codec is slave");
 		val_cr2 |= REG_SAI_CR2_MSEL_MCLK1;
 		val_cr2 |= REG_SAI_CR2_BCD_MSTR;
-		val_cr4 |= REG_SAI_CR4_FSD_MSTR;
+		val_cr2 |= SAI_CLOCK_DIV; /* TODO: determine dynamically.*/
+		break;
+	case SOF_DAI_FMT_CBM_CFM:
+		trace_sai("SAI: codec is master");
+		/*
```

Comment:
Please the trace message consistent. So, here it should be trace_sai("SAI: CBM_CFM");

Reply:
I agree that consistency it good. But I introduced a "SAI : codec is master" because it is more understandable than "SAI: CBM_CFM" and also it serves as valuable comment. The same same for the slave case.
The last two other cases are rare (bit clock and FS clock with different directions) so for consistency I can simply remove them. Would that be acceptable ?

Reply:
ok.